src/main/java/imsam/mgen/MultiTargetModelGenerator.java

```diff
@@ -0,0 +1,347 @@
+package imsam.mgen;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.json.JSONObject;
+import org.kohsuke.args4j.Option;
+
+
+/**
+ * This is a model generator that will accept a list of target states and force all paths to converge on one of said states
+ * @author Eric Reiss
+ */
+public class MultiTargetModelGenerator extends MGen{
+    public List<Integer> targets;
+    public static final String MGEN_ID = "multi-target";
```

Comment:
If this is a `public static final String`, why does it need a getter method? It is public, so anyone can access it, it is `static` so it is not even owned by the instance, and it is `final` so it won't be changed unexpectedly.

Reply:
Honestly, I am not sure. I copied what was done in the SparseModelGenerator. Maybe @gerbs-11  could explain it a little better.

Reply:
The `MGEN_ID()` method is inherited from `MGen` (it should have the `@Override` annotation). I thought making a `static final String` was a better idea than putting the literal in the function. I'm open to better ideas.